src/screens/userProfileScreens/EditUserProfileScreen.js

```diff
@@ -0,0 +1,141 @@
+import {
+	Alert,
+	Dimensions,
+	PixelRatio,
+	SafeAreaView,
+	ScrollView,
+	StyleSheet,
+	TextInput,
+	View
+} from "react-native";
+import PropTypes from "prop-types";
+import React, { useMemo, useState } from "react";
+import UserProfileAvatar from "../../components/profile/UserProfileAvatar";
+import { useDispatch, useSelector } from "react-redux";
+import GlobalStyles from "../../utils/GlobalStyles";
+import { COLOR_PALETTE } from "../../utils/constants";
+import Button, { BUTTON_TYPES } from "../../components/common/Button";
+import { launchImageLibrary } from "react-native-image-picker";
+import {
+	getUpdatedUserInformation,
+	handleUpdateProfile
+} from "../../utils/firebase";
+
+const width = Dimensions.get("window").width;
+
+const EditUserProfileScreen = ({ navigation }) => {
+	const dispatch = useDispatch();
+	const displayName = useSelector(
+		({ UserSlice }) => UserSlice?.userInformation?.displayName
+	);
+
+	const photoURL = useSelector(
+		({ UserSlice }) => UserSlice?.userInformation?.photoURL
+	);
+
+	const [editUserName, setEditUsername] = useState(
+		displayName || "Default User"
+	);
+	const [imagePreview, setImagePreview] = useState(photoURL || "");
+	const [isLoading, setIsLoading] = useState(false);
+
+	const informationNotUpdated = useMemo(() => {
+		return displayName === editUserName && photoURL === imagePreview;
+	});
+
+	const handleSelectImage = async () => {
```

Comment:
let's move this to a images util

Reply:
something like handleSelectImage(callback) where you can set-up the callback from where it's called